.github/workflows/CI.yml

```diff
@@ -31,3 +31,5 @@ jobs:
           -Dhome.application.admin.username=${{secrets.HOME_APPLICATION_ADMIN_USERNAME}}
           -Dhome.application.admin.password=${{secrets.HOME_APPLICATION_ADMIN_PASSWORD}}
           -Dapi.tests.verbose.logging=${{env.API_TESTS_VERBOSE_LOGGING}}
+          -Dmail.host=mailhog
```

Comment:
Do we really need to configure it outside of the build? What is the purpose? Can we leave it configurable and make defaults inside the pom?

Reply:
Moved to pom